### PR TITLE
Dashboard: Only allow owners to which the user belongs

### PR DESCRIPTION
**Pull request by sdford**
Also, fix local dashboard development.

### CODE REVIEW

File: dashboard/server.js [hunk 1]
```diff
@@ -6,9 +6,8 @@ var config = require("./webpack.config.js");
 var reverseProxyPort = 9001
 var nodeServerPort = 8000
 var cmsPort = 8080
-var vaultPort = 8200
 
-var loadDashboardFromCms = true
+var loadDashboardFromCms = false
```
Comment (tlisonbee) [inline dashboard/server.js]: Why this change?  Seems unrelated to the pull request.

Is false the better default setting?  I'm guessing it is better for hot-reload or similar.

Comment (sdford) [reply dashboard/server.js]: Yes, that's correct about hot-reload. If you're testing the dashboard, you want Webpack to serve the page so that you don't have to rebuild CMS every time you make a change

Comment (fieldju) [reply dashboard/server.js]: Yeah, I think it is for hot reloading, else you have to rebuild the jar and restart the webapp every time you adjust a css margin.
